core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -145,14 +154,14 @@ class AdminClient(val time: Time,
 
   def describeConsumerGroup(groupId: String): Option[List[ConsumerSummary]] = {
     val group = describeGroup(groupId)
```

Comment:
@imandhan No, I actually meant a `None` return in line 132 from `describeGroup()`. I expected to see that being handled here too (i.e. `if (!group.isDefined)`); unless I'm missing something.